main.c

```diff
@@ -431,6 +434,43 @@ uint32_t parse_color(const char *color) {
 	return res;
 }
 
+static void format_result(const char *format, char *buffer, const int buflen, struct slurp_box result) {
+	int bufferPos = 0;
+	for (int i = 0; format[i] != 0; i++) {
+		char c = format[i];
+		if (c == '%') {
+			char next = format[i + 1];
+			
+			char replacement[128] = {0};
+			switch (next) {
+				case 'x':
```

Comment:
Style: we don't indent `case` statements (`case`s have the same indentation level as `switch`)